tests/simulation/test_simulation.py

```diff
@@ -60,6 +60,7 @@ async def extra_node(self_hostname):
 
 
 class TestSimulation:
+    @pytest.mark.limit_consensus_modes(reason="This test only supports one running at a time.")
```

Comment:
This relates to https://github.com/Chia-Network/chia-blockchain/pull/16081/files#r1306648251.